keepassxc-browser/options/options.html

```diff
@@ -297,21 +297,36 @@ <h2>Ignored sites</h2>
           Sites in this list are ignored when new credentials are detected.
           <br />
           Go to the page with new credentials, click the blinking KeePassXC-Browser icon or the notification and select <em>Never ask for this page</em>.
+          <br />
+          If full ignore is enabled no input fields are detected or any credentials received for that site.
```

Comment:
Comma after "enabled" and "or any credentials received" -> "and no credentials are received"